lib/trino-filesystem-gcs/src/test/java/io/trino/filesystem/gcs/TestGcsFileSystemGcs.java

```diff
@@ -28,4 +34,16 @@ void setup()
     {
         initialize(getRequiredEnvironmentVariable("GCP_CREDENTIALS_KEY"));
     }
+
+    @Test
+    void testCreateFileRetry()
```

Comment:
how much time does it take?

Reply:
About 130 to 132 seconds.

Reply:
This is quite a lot - adjust configs to make it shorter if possible

Reply:
I did, actually the 2.0 multiplier was the minimum. Maybe I can reduce the repetitions?
When it fails it's generally before 30 tries, usually well before (7-8 repetitions). 

I will do some tests to see the min number of repetitions that guarantees we won't see flakiness.